native/explorer/src/series.rs

```diff
@@ -331,43 +341,63 @@ pub fn s_frequencies(series: ExSeries) -> Result<ExDataFrame, ExplorerError> {
 pub fn s_cut(
     series: ExSeries,
     bins: Vec<f64>,
-    labels: Option<Vec<&str>>,
+    labels: Option<Vec<String>>,
     break_point_label: Option<&str>,
     category_label: Option<&str>,
-    maintain_order: bool,
+    _maintain_order: bool,
 ) -> Result<ExDataFrame, ExplorerError> {
     let series = series.clone_inner();
-    let bins = Series::new("", bins);
-    let df = cut(
-        &series,
-        bins,
-        labels,
-        break_point_label,
-        category_label,
-        maintain_order,
-    )?;
-    Ok(ExDataFrame::new(df))
+    // Put this bool as option
+    let left_close = false;
+
+    // Cut is going to return a Series of a Struct. We need to convert it to a DF.
+    let cut_series = cut(&series, bins, labels, left_close, true)?;
+    let mut cut_df = DataFrame::from(cut_series.struct_()?.clone());
+
+    let cut_df = cut_df.insert_at_idx(0, series)?;
+
+    cut_df.set_column_names(&[
+        "values",
+        break_point_label.unwrap_or("break_point"),
+        category_label.unwrap_or("category"),
+    ])?;
+
+    Ok(ExDataFrame::new(cut_df.clone()))
 }
 
 #[rustler::nif(schedule = "DirtyCpu")]
 pub fn s_qcut(
     series: ExSeries,
     quantiles: Vec<f64>,
-    labels: Option<Vec<&str>>,
+    labels: Option<Vec<String>>,
     break_point_label: Option<&str>,
     category_label: Option<&str>,
-    maintain_order: bool,
+    _maintain_order: bool,
```

Comment:
It seems this is no longer an option right, right? So we need to remove it from the API here and in the function above.